eq_cir_proxy_service/routers/instrument_router.py

```diff
@@ -3,36 +3,41 @@
 from uuid import UUID
 
 from fastapi import APIRouter, HTTPException, Path, Query
+from structlog import get_logger
 
-from eq_cir_proxy_service.config.logging_config import logging
 from eq_cir_proxy_service.exceptions import exception_messages
-from eq_cir_proxy_service.services.instrument import instrument_retrieval_service
+from eq_cir_proxy_service.services.instrument import (
+    instrument_conversion_service,
+    instrument_retrieval_service,
+)
 from eq_cir_proxy_service.services.validators.request_validator import (
     validate_version,
 )
 from eq_cir_proxy_service.types.custom_types import Instrument
 
 router = APIRouter()
-logger = logging.getLogger(__name__)
+logger = get_logger()
 INSTRUMENT_ID_PATH = Path(..., description="UUIDv4 of the instrument")
 
 
 @router.get("/instrument/{instrument_id}")
 async def get_instrument_by_uuid(
     instrument_id: UUID = INSTRUMENT_ID_PATH,
-    version: str = Query(default=None, description="Optional version of the instrument"),
+    version: str = Query(description="Validator version of the instrument required"),
 ) -> Instrument:
-    """Retrieve an instrument by its UUID and optional version."""
+    """Retrieve an instrument by its UUID and version."""
     logger.info("Receiving the instrument id...")
-    logger.debug("Received instrument id: %s", instrument_id)
+    logger.debug("Instrument received successfully.", instrument_id=instrument_id)
```

Comment:
```suggestion
    logger.info("Instrument received successfully.", instrument_id=instrument_id)
```
Changes to `info` to represent a success status.